app/themes/light/views/layouts/_head.html.erb

```diff
@@ -24,13 +24,16 @@
 <%= favicon_link_tag "#{AppSettings['design.favicon']}" %>
 <%= cloudinary_js_config %>
 
+<% if AppSettings['settings.google_analytics_id'].present? %>
```

Comment:
This 3 snippets of code to add the GA script are all the same ? If they are, what about move it to a partial and call the partial in all places ? 

This way we can use only one conditional to check and in future if we need to add a new condition to check GA we chance this just in the partial, instead chance in 3 diff parts of code.

What do you think ?